### PR TITLE
feat(Rendering): Introduce Geometry2DRepresentation

### PR DESCRIPTION
**Pull request by sankhesh**
Depends on https://github.com/Kitware/vtk-js/pull/2179

### CODE REVIEW

**Comment by agirault** [comment]
Does react-vtk-js have some testing framework we could use here?

**Comment by jourdain** [comment]
unfortunately not. Just manual examples to test

File: usage/Geometry/PolyDataViewer.js [hunk 1]
```diff
@@ -1,46 +1,92 @@
...
+          id='1'
           property={{
-            opacity: 0.5,
+            opacity: 0.1,
             color: [0.7, 0, 0],
           }}
         >
           <PolyData
-            id="2"
+            id='2'
```
Comment (jourdain) [inline usage/Geometry/PolyDataViewer.js:28]: in general we prefer html attribute to be in `"` rather than single one.

**Review by jourdain — APPROVED** [review]
Seems reasonable. Glancing through the code seems fine.

File: usage/Geometry/PolyDataViewer.js (hunk 1 above)
Comment (sankhesh) [reply usage/Geometry/PolyDataViewer.js:28]: I see. I believe that was prettier auto-fixing it. I'll revert that.

**Comment by sankhesh** [comment]
@jourdain Could you please merge this one too?

**Comment by jourdain** [comment]
@sankhesh the pipeline is broken now...

**Comment by sankhesh** [comment]
@jourdain Thanks for letting me know. Fixing now.

**Comment by sankhesh** [comment]
Fixed in #49

**Pull request merged**